content/contributing/cs-coding-style.md

```diff
@@ -15,16 +15,22 @@ The general rule we follow is "use Visual Studio defaults". For details check th
    `namespace` declarations and should be sorted alphabetically.
 7. Avoid more than one empty line at any time. For example, do not have two
    blank lines between members of a type.
-8. Avoid spurious free spaces.
+8. Add an extra empty line after a closing bracket `}` except if another closing bracket follows or another instruction like `else` follows.
+9. Avoid spurious free spaces.
    For example avoid `if (someVar == 0)...`, where the dots mark the spurious free spaces.
    Consider enabling "View White Space (Ctrl+E, S)" if using Visual Studio, to aid detection.
-9. If a file happens to differ in style from these guidelines (e.g. private members are named `m_member`
+   Tip: use `clrl+k+d` in Visual Studio to clean the indentations, extra spaces
+10. If a file happens to differ in style from these guidelines (e.g. private members are named `m_member`
    rather than `_member`), the existing style in that file takes precedence.
-10. We only use `var` when it's obvious what the variable type is (i.e. `var stream = new FileStream(...)` not `var stream = OpenStandardInput()`).
-11. We use language keywords instead of BCL types (i.e. `int, string, float` instead of `Int32, String, Single`, etc) for both type references as well as method calls (i.e. `int.Parse` instead of `Int32.Parse`).
-12. We use PascalCasing to name all our constant local variables and fields. The only exception is for interop code where the constant value should exactly match the name and value of the code you are calling via interop.
-13. We use ```nameof(...)``` instead of ```"..."``` whenever possible and relevant.
-14. Fields should be specified at the top within type declarations.
+11. We only use `var` when it's obvious what the variable type is (i.e. `var stream = new FileStream(...)` not `var stream = OpenStandardInput()`).
+12. We use language keywords instead of BCL types (i.e. `int, string, float` instead of `Int32, String, Single`, etc) for both type references as well as method calls (i.e. `int.Parse` instead of `Int32.Parse`).
+13. We use PascalCasing to name all our constant local variables and fields. The only exception is for interop code where the constant value should exactly match the name and value of the code you are calling via interop.
+14. We use ```nameof(...)``` instead of ```"..."``` whenever possible and relevant.
+15. Fields should be specified at the top within type declarations.
+16. Always add intellisense (the /// comments) on all the public elements, including the return type, the params.
+17. Use the *exception* to describe exceptions in the intellisense comment and **avoid** adding text to the exception you raise. This will save space on the PE code. But **do** use them is contributing to the one of the tool like nanoff.
```

Comment:
This sentence seems incomplete/invalid.

> But **do** use them is contributing to the one of the tool like nanoff.

Reply:
I'm guessing this was meant to be "(...)use them when contributing to code that's not a library (like VS extension or nanoff tool) "